src/spork.h

```diff
@@ -69,16 +69,15 @@ class CSporkMessage
         READWRITE(nSporkID);
         READWRITE(nValue);
         READWRITE(nTimeSigned);
-        if (!(s.GetType() & SER_GETHASH)) {
-            READWRITE(vchSig);
-        }
+        READWRITE(vchSig);
```

Comment:
This changes the hash. Not sure if this is going to work (without being banned), will test.